config/locales/en.yml

```
@@ -8,5 +8,6 @@ en:
   api:
     errors:
       invalid_content_type: Invalid content type header
+      max_targets_reached: You have reached the maximum number (3) of targets.
```

Not a blocker, but locales supports the use of variables. You can do something like

```suggestion
      max_targets_reached: You have reached the maximum number (%{max_targets}) of targets.
```


and in the model `errors.add(:user, I18n.t('api.errors.max_targets_reached', max_targets: MAX_TARGETS))`

So that, if in the future you need to change the limit is only updating the constant 😉

---

Nice! Thanks for the comment, done to